src/components/NavBar.js

```diff
@@ -6,7 +6,7 @@ const NavBar = () => {
   return (
     <div className="nav">
       <NavLink to='/'>Home</NavLink>
-      <NavLink to='/creatures/allCR'>allCRs</NavLink>
+      {/* <NavLink to='/creatures/allCR'>allCRs</NavLink> */}
       <NavLink to='/favorites'>Favs</NavLink>
```

Comment:
I decided to cut the "allCR" component, as it's not really needed to solve the problem this app sets out to solve (i.e. letting last minute DMs find a creature by difficulty level quickly).  It would also be a hit on the fetch, as it would be pulling down all creatures from the API at once.

Reply:
That makes a lot of sense. I think it was a great idea and I like how you have a component that dynamically fetches the CR.